src/js/model/signals/index.js

```diff
@@ -6,75 +6,89 @@ var dl = require('datalib'),
 
 function api() {
   return signals;
-};
+}
+
+// Augment the signals API with properties like SELECTED that define the
+// strings used to identify and trigger a given signal
+dl.extend(api, defaults.signalNames);
 
 function ref(name) {
   return {signal: ns(name)};
 }
 
-function init(name, val) {
-  signals[name = ns(name)] = {
+api.init = function(name, val) {
+  name = ns(name);
+  signals[name] = {
     name: name,
     init: val,
     _idx: dl.keys(signals).length
   };
   return ref(name);
-}
+};
 
-function value(name, val) {
+api.getValue = function(name) {
   var model = require('../'),
+      // `view` is a vega runtime component; view.signal is a getter/setter
       view = model.view,
-      sg = signals[name = ns(name)],
-      set = arguments.length === 2;
+      signalObj = signals[ns(name)];
```

Comment:
yeah i went back and forth on that, didn't want to make a variable we never used again but i see how it could be obfuscating. I'll update.